CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Release 2.44.0 06-04-2023
+
+- Update Envoy to v1.25.4 (#809)
+- Bump API version v11 -> v12 (#808)
+- Revert: Add back in HTTP method override in service control filter (#802) (#806)
```

Comment:
remove, we didn't put original commit in last changelog

Reply:
Done.